src/main/kotlin/pt/up/fe/ni/website/backend/controller/AccountController.kt

```diff
@@ -1,23 +1,32 @@
 package pt.up.fe.ni.website.backend.controller
 
+import org.springframework.validation.annotation.Validated
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
-import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RequestPart
 import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.multipart.MultipartFile
+import pt.up.fe.ni.website.backend.annotations.validation.ValidImage
 import pt.up.fe.ni.website.backend.dto.entity.AccountDto
+import pt.up.fe.ni.website.backend.model.Account
 import pt.up.fe.ni.website.backend.service.AccountService
 
 @RestController
 @RequestMapping("/accounts")
+@Validated
```

Comment:
I'm curious, why is `@Validated` needed here?

Reply:
Validated is here so that we can use custom validators in the create acccount method, such as `@ValidImage`